src/dojoswag/DojoSwag.tsx

```diff
@@ -0,0 +1,83 @@
+import { tsx, create } from '@dojo/framework/core/vdom';
+
+import * as css from './DojoSwag.m.css';
+
+const factory = create();
+
+export default factory(function DojoShirt() {
```

Comment:
Please theme the css as a best practice.

Reply:
Done.